pages/about.vue

```diff
@@ -25,7 +25,7 @@
         {{ $t("about.supportA") }}
       </div>
       <div>
-        <h2>Who developed this site?</h2>
+        <h2>{{ $t("about.contributorQ") }}</h2>
```

Comment:
Personally, I would recommend we use more than just a letter. 
It looks like a misspell to me 😆 

We could do in a separate PR, as long as we have an issue.